examples/jsm/nodes/materials/MeshStandardNodeMaterial.d.ts

```diff
@@ -1,7 +1,23 @@
+import { Node } from '../core/Node';
+import { ColorNode } from '../inputs/ColorNode';
+import { FloatNode } from '../inputs/FloatNode';
+import { Vector2Node } from '../inputs/Vector2Node';
+import { TextureNode } from '../inputs/TextureNode';
+import { CubeTextureNode } from '../inputs/CubeTextureNode';
 import { NodeMaterial } from './NodeMaterial';
 
 export class MeshStandardNodeMaterial extends NodeMaterial {
 
 	constructor();
 
+	color: ColorNode | Node;
```

Comment:
I forgot this detail... `MeshStandardNodeMaterial ` can be accepted native and core class values too.

```js
color : Color | ColorNode | Node
roughness: number | FloatNode  | Node
metalnessMap: Texture | TextureNode | Node
```